### PR TITLE
docs: Forbid deferring tasks instead of capturing follow-ups

### PR DESCRIPTION
**Pull request by yuto-trd**
Replace the "Follow-up tasks" policy that tolerated deferring in-scope
work (capture it in the PR body or as a board Draft) with a prohibition:
finish whatever the change surfaces in the same change. The only exits are
genuinely out-of-scope or blocked items, and both require raising them with
the user rather than silently filing a follow-up.

- AGENTS.md: rewrite the section as "Do not defer work".
- PULL_REQUEST_TEMPLATE.md: add a no-deferral reminder.
- beutl-ai-review-task skill: replace Step 7 "Capture follow-ups
  (auto-register to the board)" and its Guardrails exception with the
  same prohibition.

Co-Authored-By: Claude Opus 4.8 <noreply@anthropic.com>
Claude-Session: https://claude.ai/code/session_01SSBXdjDKZTXDLg51j9z2qC

## Summary by CodeRabbit

* **Documentation**
  * Updated development workflow and process guidelines to require completing all in-scope work within the same pull request, with no deferred follow-ups.
  * Revised guidance for handling out-of-scope or blocked items, emphasizing explicit communication instead of silent deferral or project draft tracking.
  * Updated the pull request template with reminders to avoid `## Follow-ups` and ensure any exceptions are clearly raised.

### CODE REVIEW

File: .claude/skills/beutl-ai-review-task/SKILL.md [hunk 1]
```diff
@@ -264,65 +264,25 @@ gh pr create --base main --head "$BRANCH" \
...
 
-## Step 7 — Capture follow-ups (auto-register to the board)
+## Step 7 — Do not defer work
 
-If the work surfaced any follow-up — a deferred edge case, a known TODO, a refactor you scoped out,
-a test you could not add yet — **do not let it evaporate** (AGENTS.md "Follow-up tasks"). Split it
-the same way AGENTS.md does:
+**Deferring is forbidden** (AGENTS.md "Do not defer work"). Whatever this task surfaced — an edge
+case, a known TODO, a refactor you scoped out, a test you could not add yet — **finish it inside
+this PR before opening it.** Do not park in-scope work behind a `## Follow-ups` list, a `// TODO`,
```
[the remark this answers is not in the document]
Comment (yuto-trd) [reply .claude/skills/beutl-ai-review-task/SKILL.md]: Good catch. Fixed in 2d00b25 by reordering: the no-deferral check is now **Step 6**, run as a gate before the commit/push/`gh pr create` in **Step 7**. The wording no longer implies the PR must already exist.

---
_Generated by [Claude Code](https://claude.ai/code)_

[the remark this answers is not in the document]
Comment (yuto-trd) [reply .claude/skills/beutl-ai-review-task/SKILL.md]: Done in 2d00b25 — moved the no-deferral gate ahead of PR creation by swapping the steps: Step 6 is now "Do not defer work" and Step 7 is "Commit, push, PR", so the gate runs before `git push`/`gh pr create`.

---
_Generated by [Claude Code](https://claude.ai/code)_
